cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -84,6 +79,54 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
         }
     }
 
+    /**
+     * Translates an [`ExceptHandler`](https://docs.python.org/3/library/ast.html#ast.ExceptHandler)
+     * to a [CatchClause].
+     *
+     * It adds all the statements to the body and will set a parameter if it exists. For the
+     * catch-all clause, we do not set the [CatchClause.parameter].
+     */
+    private fun handleExceptHandler(node: Python.AST.ExceptHandler): CatchClause {
+        val catchClause = newCatchClause(rawNode = node)
+        catchClause.body = makeBlock(node.body, node)
+        // The parameter can have a type but if the type is None/null, it's the "catch-all" clause.
+        // In this case, it also cannot have a name, so we can skip the variable declaration.
+        if (node.type != null) {
+            // the parameter can have a name, or we use the anonymous identifier _
```

Comment:
I just had a look in the Go frontend and there, I also use an empty `name` when the name is actually empty and `_` for the anonymous identifier which is specified Go.

Specifiying `_` as the anonymous identifiers for python is actually completely wrong, since `_` is a valid identifier in python, so this creates problem! 🙈